src/mem/old_api.cc

```diff
@@ -99,6 +102,13 @@ Mem::Stats(StoreEntry * sentry)
     }
 #endif
     stream.flush();
+
+#if USE_OPENSSL
+    stream << "Current SSL memory usage:\n";
+    stream.flush();
+    Ssl::MallocStats().dump(*sentry);
+    Ssl::ReallocStats().dump(*sentry);
```

Comment:
Let's move this code inside a new `Ssl::ReportMemoryStats(StoreEntry *)` function so that this old_api.cc code does not need to know about any OpenSSL details. We might even offer to add a "call me to report my memory stats" API, but it may be an overkill, so let's start small.